src/pages/docs/infrastructure/deployment-targets/machine-policies.mdx

```diff
@@ -153,7 +153,7 @@ You can select a username/password account to perform automatic Tentacle updates
 ## Recover from communication errors with Tentacle \{#MachinePolicies-Recoverfromcommunicationerrorswithtentacle}
 
 :::div{.hint}
-Recovering from communication errors with Tentacle is available to **Cloud** customers from **Octopus 2023.3** onwards and will be made available to self-hosted customers in a future release.
+Recovering from communication errors with Tentacle is available from **Octopus 2023.4** onwards.
```

Comment:
Now that we know 2023.4 is the version, let's lock that in.

Also, everyone in Cloud has RPC retries now. So it felt strange to distinguish them here. If a Cloud customer looks at their version, they will see it's >= 2023.4.

Reply:
LGTM